### ISSUE TITLE
[Regression v8.3.0] Incorrect error: Adding route with same name or matcher as existing route.

### ISSUE THREAD

**Issue by gr2m** [wheresrhys/fetch-mock#493]
Minimal test case

```js
const fetchMock = require("fetch-mock");

function matcher1() {
  return true;
}

function matcher2() {
  return true;
}

const response = {
  ok: true
};

const mock = fetchMock
  .sandbox()
  .postOnce(matcher1, response)
  .postOnce(matcher2, response);

test();

async function test() {
  await mock("https://example.com", { method: "POST" });
  await mock("https://example.com", { method: "POST" });

  console.assert(mock.done(), "mock.done() returns true");
  console.log("done.");
}
```

In `fetch-mock` v8.2.1, it just logs `done.`. 

When running the same code in v8.3.0, it throws the following error

```
[project path]/node_modules/fetch-mock/cjs/lib/set-up-and-tear-down.js:41
                throw new Error(
                ^

Error: fetch-mock: Adding route with same name or matcher as existing route. See `overwriteRoutes` option.
    at Function.FetchMock.addRoute (/Users/gregor/Projects/octokit/auth-app.js/node_modules/fetch-mock/cjs/lib/set-up-and-tear-down.js:41:9)
    at Function.FetchMock.mock (/Users/gregor/Projects/octokit/auth-app.js/node_modules/fetch-mock/cjs/lib/set-up-and-tear-down.js:6:8)
    at Function.FetchMock.(anonymous function) [as once] (/Users/gregor/Projects/octokit/auth-app.js/node_modules/fetch-mock/cjs/lib/set-up-and-tear-down.js:78:32)
    at Function.FetchMock.(anonymous function) [as postOnce] (/Users/gregor/Projects/octokit/auth-app.js/node_modules/fetch-mock/cjs/lib/set-up-and-tear-down.js:78:32)
    at Object.<anonymous> (/Users/gregor/Projects/octokit/auth-app.js/gregor.js:18:4)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

**Comment by wheresrhys** [comment]
Thanks for the bug report. Fix on its way.

**Comment by wheresrhys** [comment]
fixed in 8.3.1
